site/content/en/docs/api_sdk/cli/_index.md

```diff
@@ -316,3 +321,25 @@ see that command's examples for more information.
   ```bash
   cvat-cli project ls --json > list_of_projects.json
   ```
+
+## Examples - functions
+
+**Note**: The functionality described in this section can only be used
+with the CVAT Enterprise or CVAT Cloud.
+
+- Create a function that uses a detection model from torchvision
+  and run an agent for it:
+
+  ```
+  cvat-cli function create-native "Faster R-CNN" \
```

Comment:
It makes sense to mention that if a custom file (module) is going to be used, the module path may require being added into `PYTHONPATH`. I'm expecting it to be a common source of questions.

Reply:
I added a note referring to the documentation of `task auto-annotate`. It has more examples, including one with `PYTHONPATH`.